### PR TITLE
Update check_domjobinfo_output function

### PR DESCRIPTION
**Pull request by cliping**

### CODE REVIEW

**Comment by cliping** [comment]
Test result in https://github.com/autotest/tp-libvirt/pull/5386.

**Review by dzhengfy — APPROVED** [review]
LGTM

File: virttest/utils_libvirt/libvirt_monitor.py [hunk 1]
```diff
@@ -53,7 +54,55 @@ def _check_items(value_type, tmp_domjobinfo):
...
+            raise exceptions.TestFail("'%s' is not matched expect '%s'" %
+                                      (sum_value, value))
+        else:
+            del tmp_domjobinfo["sum_items"][key]
+    return tmp_domjobinfo
+
+
+def check_domjobinfo_output(vm_name,
+                            expected_domjobinfo={},
+                            expected_domjobinfo_complete={},
```
Comment (Yingshun) [inline virttest/utils_libvirt/libvirt_monitor.py:102]: Plz update docstrings for these parameters, thx!

File: virttest/utils_libvirt/libvirt_monitor.py [hunk 2]
```diff
@@ -53,7 +54,55 @@ def _check_items(value_type, tmp_domjobinfo):
...
+    return None
+
+
+def _check_sum_items(line, key, tmp_domjobinfo, domjobinfo_out):
+    """
+    Check the items that need to be summed
+
+    :param line: one line of domjobinfo output
+    :param key: one key of domjobinfo output
+    :param tmp_domjobinfo: expect domain job info
```
Comment (Yingshun) [inline virttest/utils_libvirt/libvirt_monitor.py:81]: I think this function needs a special tmp_domjobinfo value to calculate the sum of some values, can you give an example in docstring so others can understand it clearly?

Comment (cliping) [reply virttest/utils_libvirt/libvirt_monitor.py]: Updated, thanks

File: virttest/utils_libvirt/libvirt_monitor.py (hunk 1 above)
Comment (cliping) [reply virttest/utils_libvirt/libvirt_monitor.py:103]: Updated, thanks.

**Review by chunfuwen — APPROVED** [review]
lgtm

**Review by Yingshun — APPROVED** [review]

**Pull request merged**